src/open_inwoner/cms/utils/middleware.py

```diff
@@ -24,3 +26,25 @@ def __call__(self, request):
                 return HttpResponseRedirect(config.redirect_to)
 
         return response
+
+
+class DropToolbarMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        if not self.use_toolbar(request):
+            request.session["cms_edit"] = False
+            request.session["cms_preview"] = False
+            request.session["cms_toolbar_disabled"] = True
```

Comment:
We might not even need these but the CMS code is really messy in when/where is looks into these so I'd just clobber them before setting the most important toolbar.show_toolbar = False